.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
```

Comment:
I'll release this change to `asciify` as a major, `2.0` release, just in case anyone is using asciify under `node` `0.8` or `0.10`